src/app/CommandHandlerImpl.cpp

```diff
@@ -623,20 +625,16 @@ CHIP_ERROR CommandHandlerImpl::FallibleAddStatus(const ConcreteCommandPath & pat
         ChipLogError(DataManagement,
                      "Endpoint=%u Cluster=" ChipLogFormatMEI " Command=" ChipLogFormatMEI " status " ChipLogFormatIMStatus " (%s)",
                      path.mEndpointId, ChipLogValueMEI(path.mClusterId), ChipLogValueMEI(path.mCommandId),
-                     ChipLogValueIMStatus(status), context);
+                     ChipLogValueIMStatus(status.GetStatus()), context);
```

Comment:
on failure, now that this is a cluster specific status code, should we also log the cluster status?

Reply:
We could

Reply:
Done